pages/pixel-ui/_meta.json

```diff
@@ -11,6 +11,11 @@
 	"formatting": "Formatting",
 	"loaders": "Loaders",
 	"scaling": "Scaling",
+	"--vgui": {
+		"title": "VGUI Reference",
+		"type": "separator"
+	},
```

Comment:
I wouldn't put this under its own separator. Instead, create a category page under the API Reference section with the name "Elements". This page should mention that elements can be created with `vgui.Create`  and include the full list of documented elements below.